packages/sdk/src/sdk/api/generated/full/apis/UsersApi.ts

```diff
@@ -2730,6 +2740,49 @@ export class UsersApi extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * @hidden
+     * Get user comment history
+     */
+    async userCommentsRaw(params: UserCommentsRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<UserCommentsResponseFull>> {
+        if (params.id === null || params.id === undefined) {
+            throw new runtime.RequiredError('id','Required parameter params.id was null or undefined when calling userComments.');
+        }
+
+        const queryParameters: any = {};
+
+        if (params.offset !== undefined) {
+            queryParameters['offset'] = params.offset;
+        }
+
+        if (params.limit !== undefined) {
+            queryParameters['limit'] = params.limit;
+        }
+
+        if (params.userId !== undefined) {
+            queryParameters['user_id'] = params.userId;
+        }
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        const response = await this.request({
+            path: `/users/{id}/comments`.replace(`{${"id"}}`, encodeURIComponent(String(params.id))),
+            method: 'GET',
+            headers: headerParameters,
+            query: queryParameters,
+        }, initOverrides);
+
+        return new runtime.JSONApiResponse(response, (jsonValue) => UserCommentsResponseFullFromJSON(jsonValue));
+    }
+
+    /**
+     * Get user comment history
+     */
+    async userComments(params: UserCommentsRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<UserCommentsResponseFull> {
```

Comment:
I maintain that it's weird that the comments endpoints don't use `get` prefixes when almost everything else does.

Reply:
That answers that question, we cool with renaming all of them?

Reply:
My preference would be to do that, but it means a major SDK version bump just to fix some names. I'm willing to let it go for now as it would just be adding unnecessary additional work.